tests/site/static/templates/test_template_redirect.py

```diff
@@ -0,0 +1,14 @@
+from tad.api.deps import templates
+from tests.constants import default_fastapi_request
+
+
+def test_tempate_redirect():
+    # given
+    request = default_fastapi_request()
+
+    # when
+    response = templates.TemplateResponse(request, "redirect.html.j2")
+
+    # then
+    assert response.headers["Content-Type"] == "text/html; charset=utf-8"
+    assert response.body == b""
```

Comment:
should we test for the header to be present?

Reply:
No, its a template